test/nxos/mocked_data/test_get_vlans/interface_without_vlans/show_vlan_brief.json

```diff
@@ -0,0 +1,261 @@
+      },
+      {
+        "vlanshowbr-vlanid": "1108",
+        "vlanshowbr-vlanid-utf": "1108",
+        "vlanshowbr-vlanname": "Vlan1108",
+        "vlanshowbr-vlanstate": "active",
+        "vlanshowbr-shutstate": "noshutdown",
+        "vlanshowplist-ifidx": "port-channel12,port-channel100,Ethernet1/21,Ethernet1/22,Ethernet1/23,Ethernet1/24,Ethernet1/47,Ethernet1/48"
+      },
+      {
+        "vlanshowbr-vlanid": "1109",
+        "vlanshowbr-vlanid-utf": "1109",
+        "vlanshowbr-vlanname": "Vlan1109",
+        "vlanshowbr-vlanstate": "active",
+        "vlanshowbr-shutstate": "noshutdown",
+        "vlanshowplist-ifidx": "port-channel12,port-channel100,Ethernet1/21,Ethernet1/22,Ethernet1/23,Ethernet1/24,Ethernet1/47,Ethernet1/48"
+      },
+      {
+        "vlanshowbr-vlanid": "1110",
+        "vlanshowbr-vlanid-utf": "1110",
+        "vlanshowbr-vlanname": "Vlan1110",
+        "vlanshowbr-vlanstate": "active",
+        "vlanshowbr-shutstate": "noshutdown",
+        "vlanshowplist-ifidx": "port-channel12,port-channel100,Ethernet1/21,Ethernet1/22,Ethernet1/23,Ethernet1/24,Ethernet1/47,Ethernet1/48"
+      },
+      {
+        "vlanshowbr-vlanid": "1111",
+        "vlanshowbr-vlanid-utf": "1111",
+        "vlanshowbr-vlanname": "Vlan1111",
+        "vlanshowbr-vlanstate": "active",
+        "vlanshowbr-shutstate": "noshutdown",
+        "vlanshowplist-ifidx": "port-channel12,port-channel100,Ethernet1/21,Ethernet1/22,Ethernet1/23,Ethernet1/24,Ethernet1/47,Ethernet1/48"
+      },
+      {
+        "vlanshowbr-vlanid": "1112",
+        "vlanshowbr-vlanid-utf": "1112",
+        "vlanshowbr-vlanname": "Vlan1112",
+        "vlanshowbr-vlanstate": "active",
+        "vlanshowbr-shutstate": "noshutdown",
+        "vlanshowplist-ifidx": "port-channel12,port-channel100,Ethernet1/21,Ethernet1/22,Ethernet1/23,Ethernet1/24,Ethernet1/47,Ethernet1/48"
+      },
+      {
+        "vlanshowbr-vlanid": "1113",
+        "vlanshowbr-vlanid-utf": "1113",
+        "vlanshowbr-vlanname": "Vlan1113",
+        "vlanshowbr-vlanstate": "active",
+        "vlanshowbr-shutstate": "noshutdown",
+        "vlanshowplist-ifidx": "port-channel12,port-channel100,Ethernet1/21,Ethernet1/22,Ethernet1/23,Ethernet1/24,Ethernet1/47,Ethernet1/48"
+      },
+      {
+        "vlanshowbr-vlanid": "1114",
+        "vlanshowbr-vlanid-utf": "1114",
+        "vlanshowbr-vlanname": "Vlan1114",
+        "vlanshowbr-vlanstate": "active",
+        "vlanshowbr-shutstate": "noshutdown",
+        "vlanshowplist-ifidx": "port-channel12,port-channel100,Ethernet1/21,Ethernet1/22,Ethernet1/23,Ethernet1/24,Ethernet1/47,Ethernet1/48"
+      },
+      {
+        "vlanshowbr-vlanid": "1115",
+        "vlanshowbr-vlanid-utf": "1115",
+        "vlanshowbr-vlanname": "Vlan1115",
+        "vlanshowbr-vlanstate": "active",
+        "vlanshowbr-shutstate": "noshutdown"
+      },
```

Comment:
VLAN 1115 does not have vlanshowplist-ifidx. I can trim out some of the extraneous VLANs/add a few more VLANs without interfaces to make the test case more apparent if that would be better

Reply:
Na, that's fine, I missed it. Thanks for clarifying!